ui/app/i18n/registration/locale_es.json

```diff
@@ -60,7 +60,7 @@
     "REGISTRATION_LABEL_BACK": "<u>E</u>spalda",
     "REGISTRATION_LABEL_ENTER_VISIT": "Introduzca\ndetalles de la <u>V</u>isita",
     "REGISTRATION_TITLE_ADDITIONAL_PATIENT": "informacion Adicional del Paciente ",
-    "REGISTRATION_TITLE_RELATIONSHIPS": "Relaciones",
+    "REGISTRATION_TITLE_RELATIONSHIPS": "Relations familiales dans l'établissement de santé",
```

Comment:
@prashantsable1991 Please revert this change as this is the spanish translation and it was correct the way it was :)